Documentation/dev/release.md

```diff
@@ -45,7 +45,9 @@ For example:
 
 ```
 unset METERING_CR_FILE
-./hack/custom-metering-operator-install-wrapper.sh release-0.8 ./hack/openshift-install.sh
+export USE_CUSTOM_METERING_OPERATOR=true
+export CUSTOM_METERING_OPERATOR_IMAGE_TAG=release-0.8
```

Comment:
should this be 0.13.0?

Reply:
No. Our release docs consistently just use 0.8 as the version for example purposes. Since it's already been released, it's more obvious this is just an example and the dev making the release should be using different values for the version they're releasing. The rest of the doc mentions this.